ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -188,7 +198,11 @@ open class PillButton: UIButton {
                     setTitleColor(style.selectedDisabledTitleColor(for: window), for: .normal)
                 }
             } else {
-                backgroundColor = style.backgroundColor(for: window)
+                if useCustomBackgroundColor {
```

Comment:
even though client override pill background color we are going to use the default selected or disabled color?

Reply:
For default(not selected) pills we were not changing the background color in disabled state.
Hence, the colors are assigned here as per style or custom.

---
In reply to: [467156825](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r467156825) [](ancestors = 467156825)